src/hidimstat/empirical_thresholding.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+from numpy.linalg import norm
+from sklearn.model_selection import GridSearchCV
+from sklearn.svm import LinearSVR
+
+
+def empirical_thresholding(
+    X,
+    y,
+    linear_estimator=GridSearchCV(
+        LinearSVR(), param_grid={"C": np.logspace(-7, 1, 9)}, n_jobs=None
+    ),
+):
+    """
+    Perform empirical thresholding on the input data and target using a linear estimator.
+
+    This function fits a linear estimator to the input data and target, and then uses the estimated coefficients to
```

Comment:
Could you make sure that the lines are no longer than 80 characters ? Long lines harm readability.